ui/src/main.rs

```diff
@@ -1176,7 +1176,24 @@ fn get_call_dialog(cx: Scope) -> Element {
                         }
                     }
                     CallDialogCmd::Reject(id) => {
-                        state.write().ui.call_info.reject_call(id);
+                        let (tx, rx) = oneshot::channel();
+                        if let Err(_e) = warp_cmd_tx.send(WarpCmd::Blink(BlinkCmd::RejectCall {
+                            call_id: id,
+                            rsp: tx,
+                        })) {
+                            log::error!("failed to send blink command");
+                            continue;
+                        }
+
+                        match rx.await {
+                            Ok(_) => {
+                                state.write().mutate(Action::RejectCall(id));
+                                state.write().ui.call_info.reject_call(id);
```

Comment:
lines 1190 and 1191 do the same thing mostly. `Action::RejectCall` results in the `state.json` file getting written to disk (because of the `state.mutate()`. `state.ui.call_info` is `#[serde(skip)]` so saving state is unnecessary. should probably delete line 1190.

Reply:
thank you, line removed